docs/csharp/language-reference/language-specification.md

```diff
@@ -34,8 +34,10 @@ translation.priority.ht:
 # C# Language Specification
 The C# Language Specification is the definitive source for C# syntax and usage. This spec contains detailed information about all aspects of the language, including many points that the documentation for Visual C# doesn't cover.  
   
- You can download this spec from the [Microsoft Download Center](http://www.microsoft.com/download/details.aspx?id=7029). If you've installed Visual Studio 2013, you can also find the spec on your computer in the Program Files (x86)/Microsoft Visual Studio 12.0/VC#/Specifications/1033 folder. However, installations of Visual Studio Express 2013 don't include this file.  
+You can download version 5.0 of this spec from the [Microsoft Download Center](http://www.microsoft.com/download/details.aspx?id=7029). If you've installed Visual Studio 2015, you can also find the spec on your computer in the Program Files (x86)/Microsoft Visual Studio 14.0/VC#/Specifications/1033 folder. However, installations of Visual Studio Express 2015 don't include this file.
+
+Version 6.0 of the spec is not currently available.
```

Comment:
nit: I was checking our internal style guide and we should never user just spec, always spell out specification. So this article wasn't following the guidelines originally.